### DISCUSSION TITLE
Supported Video Standards For Blu-Ray Video Authoring

### DISCUSSION THREAD

**Discussion by donnafaye0323** [justdan96/tsMuxer#771] [General]
As you know, following format for Blu-ray authoring are:

**720p 24fps (Blu-ray standard) , 720p 60fps and 1080i 29.97**

"You always want to go **progressive scan** as often as possible, however there is one exception to this rule... If you are shooting VERY fast motion, like sports, it may be preferable to have combing at 1080i than have judder from the slow frame rate of 24p... **The best bet on Blu-Ray for fast motion is 720p60**. Technically, you get more pixels per frame at 720p than you do 1080i, as at 60fps, 1080i is only 540 horizontal lines per frame, with every other row of pixels blank. At ****720p 60fps, you get 720 lines.''**

So I would input videos recorded in **720p at 60fps only.**

Also there's an error for joining 1080i-29.97ps videos recorded on a USB flash drive because they aren't input right. This is a Bug  and it will be fixed soon.#705  

I will help improve this program as well.

**Comment by Randrianasulu** [comment]
... May be all those discussion/brainstorming  issues can be moved to "discussion" page, if anyone know how to create one on github?

**Comment by jcdr428** [comment]
@justdan96 could you please enable the discussions on tsMuxer repository ?

**Comment by donnafaye0323** [comment]
The supported Blu-ray resolutions are 720p 24fps, 50fps and 60fps, and
1080i 24fps and 30fps. AAC audio format is not supported for Blu-ray audio.

**Comment by justdan96** [comment]
@jcdr428 Sorry for the delay - done! https://github.com/justdan96/tsMuxer/discussions
